azurelinuxagent/common/protocol/wire.py

```diff
@@ -127,25 +130,6 @@ def get_ext_handler_pkgs(self, ext_handler):
     def get_goal_state(self):
         return self.client.get_goal_state()
 
-    def _download_ext_handler_pkg_through_host(self, uri, destination):
```

Comment:
These were refactored and moved to WireClient, where the other download methods are defined